authentication/serializers.py

```diff
@@ -31,13 +31,8 @@ def validate(self, attrs):
         return attrs
 
     def create(self, validated_data):
-        user = User.objects.create(
-            first_name=validated_data['first_name'],
-            last_name=validated_data['last_name'],
-            email=validated_data['email'],
-        )
-
+        validated_data.pop('confirm_password')
+        user = super(RegisterSerializer, self).create(validated_data)
```

Comment:
Shouldn't we have to validate the serializer first and then create it?

Reply:
yes, validate method already check validation